recipes/shiba2sashimi/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+$PYTHON -m pip install . --no-deps --ignore-installed -vvv
```

Comment:
You could move this line to 'build > script' in meta.yaml and remove build.sh completely